README.md

```diff
@@ -51,6 +49,7 @@ This will display:
     mjr:~/work/node_redis (master)$
 
 Note that the API is entirely asynchronous. To get data back from the server, you'll need to use a callback.
+From v.2.6 on the API supports camelCase and snack_case and all options / variables / events etc. can be used either way.
```

Comment:
Right now I'm actually thinking about removing the documentation about the UPPER_CASE commands support as this is similar to snack_case not typical in Node.js landscape.
In general conventions as I know them this is actually even more confusing as uppercase mostly stands for constants and this is clearly not the case here.